package.json

```diff
@@ -13,14 +13,13 @@
 		"build": "dotenv -- turbo build",
 		"check": "dotenv -- turbo check:lint check:type check:format type:tests",
 		"check:format": "prettier . --check --ignore-unknown",
-		"check:lint": "dotenv -- turbo check:lint",
+		"check:lint": "node ensure-turbo-build-output.mjs && dotenv -- turbo check:lint",
```

Comment:
OOC - why put this  ensure-turbo-build-output check on here rather than say on the `check` script? Is there something special about `check:lint`?

Reply:
Nothing special about `check:link`, it just felt like the right place since it's linting turbo config files. Happy to move it though!

Reply:
Linting the turbo config files 😆  I get it. Fair enough.